source/help/apps/desktop-mvp.rst

```diff
@@ -0,0 +1,9 @@
+**The following awards have been given for significant contributions to the Mattermost Desktop App:**
```

Comment:
We already have a 'credits' page here: https://www.mattermost.org/team/

I feel this should be separated as it includes the MVP per release, not just a list of contributors.